sdk/ai/ai-inference-rest/samples-dev/chatCompletions.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates how to get chat completions for a chat context.
+ *
+ * @summary get chat completions.
+ */
+
+import ModelClient, { isUnexpected } from "@azure-rest/ai-inference";
+import { AzureKeyCredential } from "@azure/core-auth";
```

Comment:
Could we use token credential instead?

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-js/pull/30211/commits/8a5b2d674c7ec4d3fc2b095b3208a728400a73e4)